src/main/java/com/powsybl/openloadflow/ac/outerloop/IncrementalTransformerVoltageControlOuterLoop.java

```diff
@@ -56,14 +57,32 @@ public String getName() {
         return NAME;
     }
 
+    public static List<LfBus> getControlledBuses(LfNetwork network) {
+        return network.getBuses().stream()
+                .filter(bus -> bus.isVoltageControlled(VoltageControl.Type.TRANSFORMER))
+                .filter(bus -> bus.getVoltageControl(VoltageControl.Type.TRANSFORMER).orElseThrow().getMergeStatus() == VoltageControl.MergeStatus.MAIN)
+                .filter(bus -> !bus.getVoltageControl(VoltageControl.Type.TRANSFORMER).orElseThrow().isHidden(true))
+                .collect(Collectors.toList());
+    }
+
+    public static List<LfBranch> getControllerElements(LfNetwork network) {
+        return network.getBuses().stream()
+                .filter(bus -> bus.isVoltageControlled(VoltageControl.Type.TRANSFORMER))
+                .filter(bus -> bus.getVoltageControl(VoltageControl.Type.TRANSFORMER).orElseThrow().getMergeStatus() == VoltageControl.MergeStatus.MAIN)
+                .filter(bus -> !bus.getVoltageControl(VoltageControl.Type.TRANSFORMER).orElseThrow().isHidden(true))
+                .flatMap(bus -> bus.getTransformerVoltageControl().orElseThrow().getMergedControllerElements().stream())
+                .filter(Predicate.not(LfBranch::isDisabled))
+                .collect(Collectors.toList());
+    }
```

Comment:
Same comment as for `IncrementalShuntVoltageControlOuterLoop `.

Also some code mutualization between `IncrementalShuntVoltageControlOuterLoop` 
 and `IncrementalTransformerVoltageControlOuterLoop ` would be welcomed here.